pkg/labels/labels.go

```diff
@@ -71,6 +71,8 @@ const (
 	PrometheusScrape = "prometheus.io/scrape"
 	PrometheusPath   = "prometheus.io/path"
 	PrometheusPort   = "prometheus.io/port"
+
+	CiliumProxyVisibility = "policy.cilium.io/proxy-visibility"
```

Comment:
Think its worth using this package?

https://github.com/cilium/cilium/blob/696167cb625e3b61b42412a37f7b02adcce2ff3f/pkg/annotation/k8s.go#L107

Reply:
I just tried it out but there are multiple mutual dependencies between us and Cilium, so adding this updated some of those and I'm a little afraid of causing a problem and would just prefer to have this in our code.